test/e2e/driver/azuredisk_csi_driver.go

```diff
@@ -26,35 +28,52 @@ import (
 	storagev1 "k8s.io/api/storage/v1"
 	"k8s.io/apimachinery/pkg/api/resource"
 	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/klog"
 )
 
+const AzureDriverNameVar = "AZURE_STORAGE_DRIVER"
+
 // Implement DynamicPVTestDriver interface
 type azureDiskCSIDriver struct {
 	driverName string
 }
 
+func normalizeProvisioner(provisioner string) string {
```

Comment:
why we need this? provide a func comment?

Reply:
I think it's because storage class name cannot contain `/` character, but in-tree provisioner contains `/` from `kubernetes.io/azure-disk`.

Reply:
sounds reasonable, pls put a comment in the function, other part LGTM